src/cli/templates.hh

```diff
@@ -1659,10 +1656,7 @@ headers = native-module1.hh
 ; cmd = "node backend/index.js"
 
 ; The icon to use for identifying your app on Windows.
-icon = ""
-
-; The icon to use for identifying your app on Windows.
-logo = "src/icons/icon.png"
```

Comment:
legacy? changed all `win_logo` to be `win_icon`

Reply:
win_logo was for the app package manifest, win_icon is the actual icon of the app, there may be a better name for it

Reply:
I brought `win_logo` back to not block this PR, we can change it's name later if necessary

Reply:
`[mac] appstore_icon` is not used anywhere so I removed it until we need it